lib/glific/partners.ex

```diff
@@ -354,6 +355,23 @@ defmodule Glific.Partners do
     end
   end
 
+  @doc """
+  Returns quality rating information for an organization provider
+  """
+  @spec get_quality_rating(pos_integer()) :: {:ok, any()} | {:error, String.t()}
+  def get_quality_rating(organization_id) do
+    organization = organization(organization_id)
+
+    if is_nil(organization.services["bsp"]) do
+      {:error, dgettext("errors", "No active BSP available")}
+    else
+      case organization.bsp.shortcode do
+        "gupshup" -> PartnerAPI.get_quality_rating(organization.contact.phone)
```

Comment:
let's send org_id in the partner API and fetch the phone there. Just like gupshup messages, so that in case we need more information in future we can get it from org_id itself.